gremlin-javascript/src/main/javascript/gremlin-javascript/.eslintrc.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     node: true,
   },
   parserOptions: {
-    ecmaVersion: 2017,
+    ecmaVersion: 2020,
```

Comment:
is this a breaking change?

Reply:
I bumped the version on eslint as it was blocking the use of the ?? operator which has been available since node 14. It was my understanding that tinkerpop required node >= 16. Upon further investigation, package.json node only needs to be >=10 for tinkerpop. I will revert this change and find a replacement for ?? wherever I've used it.